ray-operator/controllers/ray/utils/util.go

```diff
@@ -201,10 +201,13 @@ func CalculateMaxReplicas(cluster *rayiov1alpha1.RayCluster) int32 {
 	return count
 }
 
-// CalculateDesiredReplicas calculate desired worker replicas at the cluster level
+// CalculateAvailableReplicas calculates available worker replicas at the cluster level
 func CalculateAvailableReplicas(pods corev1.PodList) int32 {
 	count := int32(0)
 	for _, pod := range pods.Items {
+		if val, ok := pod.Annotations["ray.io/node-type"]; !ok || val != string(rayiov1alpha1.WorkerNode) {
```

Comment:
`ray.io/node-type` is a label rather than an annotation.

<img width="649" alt="Screen Shot 2023-01-29 at 7 33 14 AM" src="https://user-images.githubusercontent.com/20109646/215336908-a427bae5-beda-4236-b984-f74b449b732c.png">

Reply:
yes thanks